### ISSUE TITLE
"Loaded" announce does not display the correct time

### ISSUE THREAD

**Issue by Phentora** [cydrobolt/nullchat#6]
What is the issue: The time from the "Loaded" announce is 00:00:00.

What should happen: The announce should post the time it actually loaded the room.

Issue tested on Google Chrome 55.0.2883.87 m (64-bit) and Internet Explorer 11.0.9600.18537.

**Comment by cydrobolt** [comment]
Yeah, I agree. Currently, the "loaded" text is static and is displayed to fix the layout. Perhaps we can figure out how to display the correct time instead of leaving it at 00:00:00, but it's not a big priority :)

**Comment by Phentora** [comment]
Yup. I just figured out it's nice to leave this issue submitted here.
